screwdriver.yaml

```diff
@@ -22,14 +22,14 @@ jobs:
     - save_version: cat $SD_ARTIFACTS_DIR/publish.output | awk -F[@\ ] '{print $3}' | tee $SD_ARTIFACTS_DIR/template_version
     - tag_latest: template-tag --name screwdriver-cd/semantic-release --version `cat $SD_ARTIFACTS_DIR/template_version` --tag latest
     - tag_stable: template-tag --name screwdriver-cd/semantic-release --version `cat $SD_ARTIFACTS_DIR/template_version` --tag stable
-  publish-node8:
-    requires: [main]
-    image: node:8
-    environment:
-      SD_TEMPLATE_PATH: semantic_release_template_node8.yaml
-    steps:
-    - install_binaries: npm i -g screwdriver-template-main
-    - publish_template: template-publish | tee $SD_ARTIFACTS_DIR/publish_node8.output
-    - save_version: cat $SD_ARTIFACTS_DIR/publish_node8.output | awk -F[@\ ] '{print $3}' | tee $SD_ARTIFACTS_DIR/template_version_node8
-    - tag_latest: template-tag --name screwdriver-cd/semantic-release-node8 --version `cat $SD_ARTIFACTS_DIR/template_version_node8` --tag latest
-    - tag_stable: template-tag --name screwdriver-cd/semantic-release-node8 --version `cat $SD_ARTIFACTS_DIR/template_version_node8` --tag stable
+  # publish-node8:
```

Comment:
Can we remove commented out code/template?

Reply:
There is a template for node8 deployed separately, so I did not delete it.